harvester/utils.py

```diff
@@ -0,0 +1,21 @@
+"""utils.py module."""
+
+import sentry_sdk
+
+
+def send_sentry_message(
+    message: str,
+    scopes: dict | None = None,
+    level: str = "warning",
+):
+    """Send message directly to Sentry.
+
+    This allows both reporting information without raising an Exception, and optionally
+    including additional information in the sent Sentry message via "scopes".
```

Comment:
Description of the args would be helpful, `scopes` isn't very clear on its own

Reply:
Good call!  Done.